### ISSUE TITLE
Accessibility issues - non conforming WCAG, ADA and others guidelines

### ISSUE THREAD

**Issue by eduardaloureiro** [status-im/status-react#17778]
# Bug Report
The app design does not comply with accessibility guidelines. The app overall is not accessible.

## Problem 
[comment]: # (A problem description)
Accessibility is not present into the app, general concepts such as color contrast, spacing, font size and others are not being considered, going against the laws.

#### Expected behavior
A possible redesing to fix the accessibility issues or provide an feature/switcher that apply into the app when enabled.

#### Actual behavior
Not accessible. 

### Reproduction

1.  n/a

### Additional Information

- Status version: nightly 
- Operating System: Android, iOS - both.

[comment]: # (Please, add logs/notes if necessary)

**Comment by eduardaloureiro** [comment]
Here devs and PO can find basics tips to reach accessibility basic guides: https://www.youtube.com/watch?v=mojEOA-QeKY

**Comment by ilmotta** [comment]
My belated thanks, @eduardaloureiro!

I strongly agree with this GH issue and have expressed my concerns a few times in the past. Realistically, it will have to wait even longer to be addressed, as Status needs to focus on improving other critical features first.
